lib/validators/custom/confirmed_test.go

```diff
@@ -0,0 +1,41 @@
+package custom_test
+
+import (
+	"github.com/markgravity/golang-ic/lib/validators"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("#ConfirmedValidator", func() {
+	Context("given VALID payload", func() {
+		It("does NOT return any errors", func() {
+			validators.Init()
+			payload := struct {
+				Password             string `binding:"confirmed"`
```

Comment:
```suggestion
				Email             string `binding:"confirmed"`
```

Should we try with another field than the one we already use? To confirm the generic side of this validator? :)

Reply:
@malparty I think using Password here is great example to show how we use this validator, email is quite weird when using with `confirmed` IMO 🤔

Reply:
https://formcraft-wp.com/help/how-to-make-a-confirm-email-field/

You'll be surprised that it's still a practice 🙈 

But that's not the point. You've built the validator so that is it not limited to the Passport field (i.e. it is a generic validator). This is nice and elegant 👍. But the test does NOT cover the fact that the validator can be used for any other field.

If `email` is weird, we can use "AnyFieldName" to be really generic. And if the "how to use" is not clear enough, we might need to change:
- validator name (confirmable_field?)
- test description: "given 2 fields named '{name}' and '{name}confirmed'"
- ...

But the point is to ensure the test is generic and not limited to the current usage; in which case we could have implemented a `passwordConfirmed` validator that hardcodes the value of the field names :)

Reply:
@malparty solved at [6fe83d8](https://github.com/markgravity/golang-ic/pull/38/commits/6fe83d87d767236af7c1560ce7c1f9a1557a42ae)